jwql/instrument_monitors/common_monitors/readnoise_monitor.py

```diff
@@ -385,17 +388,18 @@ def most_recent_search(self):
             Date (in MJD) of the ending range of the previous MAST query
             where the readnoise monitor was run.
         """
-
-        query = session.query(self.query_table).filter(and_(self.query_table.aperture == self.aperture,
-                self.query_table.run_monitor == True)).order_by(self.query_table.end_time_mjd).all()  # noqa: E712 (comparison to True)
+        filter_kwargs = {
+            'aperture__iexact': self.aperture,
+            'run_monitor__exact': True
+        }
+        query = self.query_table.objects.filter(**filter_kwargs).order_by("-end_time_mjd").all()
 
         if len(query) == 0:
             query_result = 59607.0  # a.k.a. Jan 28, 2022 == First JWST images (MIRI)
             logging.info(('\tNo query history for {}. Beginning search date will be set to {}.'.format(self.aperture, query_result)))
         else:
-            query_result = query[-1].end_time_mjd
+            query_result = query[0].end_time_mjd
```

Comment:
So here we are trying to get the last time the monitor was run, right? Instead of ordering chronologically by time and grabbing the last item in the list using `-1` indexing we sorting in reverse chronological order and grab the first item in the list?